lightning/src/ln/channelmanager.rs

```diff
@@ -8737,6 +8737,43 @@ where
 	#[cfg(c_bindings)]
 	create_refund_builder!(self, RefundMaybeWithDerivedMetadataBuilder);
 
+	fn create_invoice_request_messages(&self, invoice_request: InvoiceRequest, reply_path: BlindedPath)
```

Comment:
Let's move this under `pay_for_offer` to keep the reading top-down.

Reply:
Done! [fa5cf60](https://github.com/lightningdevkit/rust-lightning/commit/fa5cf6003f56940682f578688cd33c68838086e9)